modules/crud/sql/describe.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/spaceuptech/space-cloud/utils"
 )
 
-// not to be exposed externally
+// NOTE: not to be exposed externally
```

Comment:
Comments are wrong. Always start with the function name you are describing. Everything else goes below that